package.json

```diff
@@ -117,6 +117,11 @@
       "name": "Luis Orbaiceta",
       "email": "luisorbaiceta@gmail.com",
       "url": "https://luisorbaiceta.com"
+    },
+    {
+      "name": "Jesús Leganés-Combarro 'piranna'",
+      "email": "jesus.leganes.combarro@gmail.com",
+      "url": "https://piranna.github.io"
```

Comment:
List is for core members of org

```suggestion
```

Reply:
Sorry for that, I though it was for all contributors, reverted.